OperatorAssignment2.js

```diff
@@ -0,0 +1,48 @@
+// **********************************  JavaScript Task #2 Operator Problem  ***********************************************
+
+const array  = ["OR",["<","a","b"],["AND",["==","c","d"],["!=","e","f"]]]
+
+const flatedArray = array.flat(Infinity); //removing nested arrays
+let length = flatedArray.length;
+let i = length-1;
+console.log('\n');
+let bufferArray=[];
+let buff2=[];
+let j=1;
+while( i >= 0 ){ 
+    if( bufferArray.length === 3){
+        let firstElement = bufferArray[bufferArray.length -1];
```

Comment:
Same as in the last file (OperatorAssignment.js)
- let firstElement =  bufferArray.pop()
- let secondElement = bufferArray.pop()